themes/10up-block-theme/includes/blocks/movie-release-year/edit.js

```diff
@@ -0,0 +1,29 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import { useEntityProp } from '@wordpress/core-data';
+import { __ } from '@wordpress/i18n';
+import { usePost } from '@10up/block-components';
+
+export const BlockEdit = () => {
```

Comment:
I would actually make this editable inline. When the block is then used in a template we can make it editable using the following filter:

```js
addFilter('editor.postContentBlockTypes', 'namespace/example', (blockTypes) => {
	return [...blockTypes, 'tenup/example-block'];
});
```